Jenkinsfile

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright © 2022, California Institute of Technology ("Caltech").
+ * U.S. Government sponsorship acknowledged.
+ * 
+ * All rights reserved.
+ * 
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * • Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ * • Redistributions must reproduce the above copyright notice, this list of
+ * conditions and the following disclaimer in the documentation and/or other
+ * materials provided with the distribution.
+ * • Neither the name of Caltech nor its operating division, the Jet Propulsion
+ * Laboratory, nor the names of its contributors may be used to endorse or
+ * promote products derived from this software without specific prior written
+ * permission.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+// Declarative Pipeline
+// ====================
+//
+// This is a Jenkins pipline (of the declarative variety) for continuous deployment of the Registry.
+// See https://www.jenkins.io/doc/book/pipeline/syntax/ for more information.
+
+pipeline {
+
+    // We want this to run completely on pds-expo.jpl.nasa.gov and nowhere else
+    agent { node('pds-expo') }
+
+    environment {
+        // Pipeline-Specific Environtment
+        // ------------------------------
+        //
+        // How long to wait (in seconds) before killing containers:
+        shutdown_timeout = "30"
+        // Simplified `docker-compose` command:
+        compose = "docker-compose --profile int-registry-batch-loader --project-name registry --file ${env.WORKSPACE}/docker/docker-compose.yml"
```

Comment:
We used `docker compose` instead of `docker-compose` in all other places. Can we use `docker compose` here also for consistency?

Reply:
Sadly no. The version of Docker on `pds-expo` does not support an integrated `compose`. It must be `docker-compose`.

But sharp eyes!